nevergrad/optimization/optimizerlib.py

```diff
@@ -1668,3 +1717,36 @@ def __init__(self, parametrization: IntOrParameter, budget: Optional[int] = None
                     self.optims = [CMA(self.parametrization, budget, num_workers)]
             else:
                 self.optims = [NGO(self.parametrization, budget, num_workers)]
+
+
+@registry.register
+class MetaModel(base.Optimizer):
+    """Adding a metamodel into CMA."""
+
+    def __init__(self, parametrization: IntOrParameter, budget: Optional[int] = None, num_workers: int = 1) -> None:
```

Comment:
Does that work for any optimizer? you may add a parameter for stating the base optimizer name, then make a Configured optimizer to specify it:
Eg: MetaModel("CMA")(parametrization, budget, num_workers)

Reply:
It works for all optimizers  ---  the underlying optimizer is now a parameter.